pkg/issuer/acme/dns/util/wait_test.go

```diff
@@ -54,7 +54,7 @@ var findZoneByFqdnTests = []struct {
 	zone string
 }{
 	{"mail.google.com.", "google.com."},             // domain is a CNAME
-	{"foo.google.com.", "google.com."},              // domain is a non-existent subdomain
+	{"foo.google.com.", "google.com."},              // domain is a nonexistent subdomain
```

Comment:
Note that this isn't legal and is subject to break (see earlier RFC reference).

cert-manager should pick some domains it actually controls and use them instead.

Reply:
Let's maybe fix this later?

Reply:
- #7578 so we don't forget